.github/workflows/query-engine.yml

```diff
@@ -10,6 +10,7 @@ on:
       - 'LICENSE'
       - 'CODEOWNERS'
       - 'renovate.json'
+      - 'introspection-engine/**'
```

Comment:
That is an interesting addition - avoiding running the QE tests on IE only changes I assume?

Reply:
Yes, not reflected but it seems like a small win.